### PR TITLE
Format address

### PR DESCRIPTION
**Pull request by pivilartisant**
Screenshot from 2023-07-06 15-12-48

Screenshot from 2023-07-06 15-12-52

### CODE REVIEW

**Comment by Pierregvx** [comment]
can you show a screenshot ?

File: wails-frontend/src/pages/backupKeyPairs.tsx [hunk 1]
```diff
@@ -117,6 +117,7 @@ function BackupKeyPairs() {
                 <Clipboard
                   displayedContent={publicKey}
                   rawContent={publicKey as string}
+                  toggleHover={false}
```
Comment (maxime-carabina) [inline wails-frontend/src/pages/backupKeyPairs.tsx:120]: For what you set `toggleHover` to `false` ?

Comment (pivilartisant) [reply wails-frontend/src/pages/backupKeyPairs.tsx:120]: @SlnPons and I  (for multiple product related reasons) have decided to remove the toggle feature, but keep it in the code base. So we only remove it from components but it stays in the ui-kit.

Comment (maxime-carabina) [reply wails-frontend/src/pages/backupKeyPairs.tsx:120]: > @SlnPons and I (for multiple product related reasons) have decided to remove the toggle feature, but keep it in the code base. So we only remove it from components but it stays in the ui-kit.

OK, but if you decide to set it to Off, that means that there is no default value in the UI-KIT.

Unless you've made sure that when you don't use the 'toggleHover' value, there's no hover.

**Review by Pierregvx — APPROVED** [review]

File: wails-frontend/src/pages/backupKeyPairs.tsx (hunk 1 above)
Comment (maxime-carabina) [reply wails-frontend/src/pages/backupKeyPairs.tsx:120]: We will do this update in a new PR

**Review by maxime-carabina — APPROVED** [review]

**Pull request merged**